scripts/export_models_ops_correlation.py

```diff
@@ -70,6 +70,7 @@ def run_and_export_models_unique_ops_configuration(compilation_depth, pytest_dir
                     FORGE_COMPILE_DEPTH=compilation_depth,
                     FORGE_EXTRACT_UNIQUE_OP_CONFIG_AT=compilation_depth.upper(),
                     FORGE_EXPORT_UNIQUE_OP_CONFIG_TO_CSV="1",
+                    FORGE_EXPORT_UNIQUE_OP_CONFIG_TO_XLSX="1",
```

Comment:
Is it better to have only one env variable `FORGE_EXPORT_UNIQUE_OP_CONFIG_TYPE` and its value being "csv" or "xslx"? This is just to try having the least amount of env variables...

Reply:
The script used for cross correlating models variants and ops and sheet creation for models unique op configuration is currently using the csv file as input and xlsx are dumped in addition if someone want to see each  model unique configurations separately. If we are using one env variable, for generating sheet they need to run script with csv value for env variable and for better visualization as xlsx file, they need to run script with xlsx value for env variable and then dump the unique op configuration.
I will try to check if I am able to use xlsx file as input for generating sheet that contains cross correlation between model varaints and unique op configs. If yes, I will try to use single env variable.

Reply:
Modified the code for dumping either as csv or xlsx file by having one env variable. 
New env Flag:
`FORGE_EXPORT_UNIQUE_OP_CONFIG_FILE_TYPE = {csv/xlsx}`
The script also has the option to process either as csv or xlsx file.